### DISCUSSION TITLE
modify ModLoot.java to enable modded structures loot tables injection

### DISCUSSION THREAD

**Discussion by aria1th** [TechReborn/TechReborn#2776] [Ideas]
https://github.com/aria1th/TechReborn/blob/1.18/src/main/java/techreborn/init/ModLoot.java

Now its hardcoded to only inject to vanilla structures. Mods like repurposed structures can't allow injection due to this.
Hacky solution is to use contains() method or something else... but ultimate answer should be whitelist / blacklist to loot tables entry.

**Comment by Ayutac** [comment]
Whitelist: You mean a json file that lists to which structures the loot is added to?

**Reply by aria1th** [reply]
maybe, it should be discussed because I have no idea if it should be whitelist/ blacklist / or just json table that defines all structures : table...
maybe something like this :
{"default" : "basic_loot",
"modA:no" : "none",
"modB:important" : "advanced_loot",
"modC:veryimportant" : "industrial_loot"}

**Reply by Ayutac** [reply]
I mean, any other mod could include our loot pools just  as well as we can include their structure. If you have specific structure loot chests in mind, we can always directly add it

**Reply by aria1th** [reply]
there's so many structure mods such as repurposed structures / sky villages, and even adventure mods... and some structure mods even tries to overwrite vanilla ones so hmm... maybe repurposed structures as their download count?
